identifier/src/containers/Login/Login.jsx

```diff
@@ -81,7 +81,13 @@ function Login(props) {
       }
 
       history.replace(`/chooseaccount${history.location.search}${history.location.hash}`);
-      return;
+        return;
+      }
```

Comment:
Why is this so hard @soun059 - indentation errors are easy to see if you look at your own MR.

Reply:
Updated the indentation properly. The return was getting an extra space which was not visible as new changes. Removed the formatter from editor helped to understand why this extra space was getting added.